packages/strapi-helper-plugin/lib/src/app.js

```diff
@@ -8,11 +8,25 @@
 import React from 'react';
 import { Provider } from 'react-redux';
 
-import App, { bootstrap } from 'containers/App'; // eslint-disable-line
+import App from 'containers/App'; // eslint-disable-line
 
 import configureStore from './store';
 import { translationMessages } from './i18n';
 
+const tryRequire = (bootstrap = false) => {
+  try {
+    const path = 'bootstrap';
+    const config = bootstrap ? require('bootstrap').bootstrap : require('pluginRequirements').shouldRenderCompo;
+    return config;
```

Comment:
Use a dynamic return: `return require(fileName);`

Reply:
Can't do that  since passing a fileName parameter didn't work